src/overlays/actors/ovl_En_Bubble/z_en_bubble.h

```diff
@@ -10,7 +11,25 @@ typedef void (*EnBubbleActionFunc)(struct EnBubble*, PlayState*);
 typedef struct EnBubble {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnBubbleActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x110];
+    /* 0x148 */ ColliderJntSph colliderSphere;
+    /* 0x168 */ ColliderJntSphElement colliderElements[2];
+    /* 0x1F4 */ Vec3f unk1F4; // set but never used
+    /* 0x1F4 */ Vec3f unk1F8; // randomly generated, set but never used
+    /* 0x200 */ s16 unk200; // set to 8 when about to pop
```

Comment:
Style for unks is meant to include a `_`. There are a lot without, but far more with now.